Puppetfile

```diff
@@ -27,12 +27,13 @@ mod 'puppetlabs-puppet_conf', '0.4.0'
 mod 'puppetlabs-python_task_helper', '0.3.0'
 mod 'puppetlabs-reboot', '2.2.0'
 mod 'puppetlabs-ruby_task_helper', '0.4.0'
+mod 'puppetlabs-ruby_plugin_helper', '0.1.0'
 
 # Plugin modules
 mod 'puppetlabs-azure_inventory', '0.2.0'
 mod 'puppetlabs-terraform', '0.2.0'
 mod 'puppetlabs-vault', '0.2.2'
-mod 'puppetlabs-aws_inventory', '0.2.0'
+mod 'puppetlabs-aws_inventory', '0.3.0'
```

Comment:
I think it would make sense to release all the `inventory` modules with `target_mappings` at the same time/release. That we we can definitively say: these require `target_mappings`.